README.md

```diff
@@ -281,3 +283,53 @@ var editor = EditorJS({
   ...
 });
 ```
+
+## Providing custom image url resolving methods
+
+As mentioned at the Config Params section, you have an ability to provide own custom image resolving method.
+It is a quite simple: implement `resolveUrlByFileData` method and pass it via `imageResolver` config param.
+Method must return a Promise that resolves with url, which we can pass to the image element and show it
+
+
+| Method         | Arguments | Return value | Description |
+| -------------- | --------- | ------------- | ------------|
+| resolveUrlByFileData       | `fileData`| `Promise.<string>` | Resolve image url by file data |
+
+`fileData` - any data, which is needed for resolving url
```

Comment:
is it an any data returned from the server?

Reply:
updated desc